swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md

```diff
@@ -199,7 +199,7 @@ Now, you tested a simple Ballerina HTTP server application for GraalVM compatibi
 
 ## Create GraalVM-compatible library packages
 
-Packages that solely use Ballerina standard libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
+Packages that solely use Ballerina libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
```

Comment:
It is incorrect to Ballerina Libraries. It is a singular thing. Refer to https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#discussion_r1328880832

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834.